webapp/channels/src/components/integrations/bots/bots.tsx

```diff
@@ -260,11 +262,16 @@ export default class Bots extends React.PureComponent<Props, State> {
                                 ),
                             }}
                         />
-                        <FormattedMarkdownMessage
-                            id='bots.manage.help2'
-                            defaultMessage={'Enable bot account creation in the [System Console]({siteURL}/admin_console/integrations/bot_accounts).'}
+                        <FormattedMessage
+                            id='bots.help2'
+                            defaultMessage={'Enable bot account creation in the <a>System Console</a>.'}
                             values={{
-                                siteURL: getSiteURL(),
+                                a: (chunks: string) => (
+                                    <a
+                                        href={`${getSiteURL()}/admin_console/integrations/bot_accounts`}
```

Comment:
if we use the Link component directly we dont need siteURL, since the router will construct it for you

Reply:
Aah! Thanks I knew I was missing an internal link component somewhere, will fix that. @M-ZubairAhmed